MTM-61027 - RN: Enabling time series support on management tenant is no longer possible (#2484)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-10-20-584-0-enabling-time-series-on-management-no-longer-possible.md

---

> date: 2024-10-07

Reviewer: ```suggestion
date: 
```

Reviewer: The date will be synchronized with the deployment of the version on Zone 1.

> title: Enabling time series support on management tenant is no longer possible

Reviewer: ```suggestion
title: Enabling time series support on the {{< management-tenant >}} is no longer possible
```

> title: Enabling time series support on the {{< management-tenant >}} is no longer possible

Reviewer: ```suggestion
title: Enabling time series support on the Management tenant is no longer possible
```

product_area: Platform services
change_type:
  - value: change-3BQrQ6adS

>     label: API change

Reviewer: @mbrz-c8y Is this a breaking change? Otherwise please use a different label (for example, improvement).

Author: Technically, this change alters the behavior of the tenant option API but maybe we should just consider it as a bugfix. @martin-c8y, @pkepa7, @jaro-herod what do you think?

Reviewer 2: I would consider it exactly a bugfix. This switch is not needed on management tenant, and normal customers should not store there anything. I am not sure if we need to have RN here, or only in c8y-docs-ops.

Reviewer: @mbrz-c8y Then please change the label and the value accordingly.

Author: I changed it to `Fix`

Reviewer: @mbrz-c8y Can be merged from my side

component:
  - value: component-OG_650_b2
    label: Core platform
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-61027
version: 10.20.584.0
---

> Enabling [time series support](https://cumulocity.com/docs/enterprise-tenant/enhanced-time-series-support/) on `management` tenant is no longer possible to prevent time series from being enabled on sub-tenants by tenant option inheritance.

Reviewer: ```suggestion
Enabling [time series support](https://cumulocity.com/docs/enterprise-tenant/enhanced-time-series-support/) on the {{< management-tenant >}} is no longer possible to prevent time series from being enabled on subtenants by tenant option inheritance.
```
